schematic/manifest/generator.py

```diff
@@ -1586,10 +1586,10 @@ def get_manifest(
                 manifest_url, manifest_df = self.get_manifest_with_annotations(annotations)
```

Comment:
Note that here, we are assuming annotations can't contain protected, e.g. PHI, contributors' data. Otherwise, we'd be communicating that with Google.

Reply:
I think this is fine for now; something to highlight with ISO and see if it'd be a problem.